commands/build.go

```diff
@@ -89,12 +90,15 @@ func runBuild(cmd *cobra.Command, args []string) error {
 		build(&services, parallel, shrinkwrap)
 	} else {
 		if len(image) == 0 {
+			fmt.Println("Please provice a valid -image name for yout Docker image")
```

Comment:
`provide` is spelt incorrectly.
`your` is spelt incorrectly.

Also should be using the posix style flag here in the message `--image` not the go style.